squape/settings.py

```diff
@@ -25,6 +26,9 @@ def _ctx_settings(setting_name, value):
     """
     try:
         current_value = getattr(squish.testSettings, setting_name)
+        test.log(
```

Comment:
I would prefer to use here report.debug(msg), from squape.report. This would give the user option to decide wheater to see those entries or not.